### PR TITLE
Update MTK compat bound and minor release

### PR DESCRIPTION
**Pull request by AlCap23**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ DataInterpolations = "3.1"
 DiffEqBase = "6.45"
 DocStringExtensions = "0.7, 0.8"
 FiniteDifferences = "0.11"
-ModelingToolkit = "4.0.8"
+ModelingToolkit = "4"
```
Comment (ChrisRackauckas) [inline Project.toml:28]: any reason to drop the patch requirement?

Comment (AlCap23) [reply Project.toml:28]: Just experimenting around and trying to be more flexible wrt the upper bound given MTKs release speed. Possibly `ModelingToolkit = "^4.0.8"`
is the better choice here, but I clicked to fast :smile:

Comment (ChrisRackauckas) [reply Project.toml]: it's implicitly ^4.0.8 already.

Comment (AlCap23) [reply Project.toml]: Ah, alright. Thanks :). Than I will remove this PR and just prepare a minor release.

**Pull request closed without merging**